controller/array_action/array_mediator_interface.py

```diff
@@ -66,6 +66,7 @@ def copy_to_existing_volume_from_snapshot(self, name, src_snap_name, src_snap_ca
             src_snap_capacity_in_bytes   : capacity of snapshot to create from
             min_vol_size_in_bytes        : if snapshot capacity is lower than this value vol will
                                            be increased to this value
+            pool: pool of the volume and snapshot to find them more efficiently.
```

Comment:
consider renaming `pool` to `pool_id`/`pool_name` (whichever is more accurate, wherever applicable)